app/mailers/people/neuanmeldungen_mailer.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+class People::NeuanmeldungenMailer < ApplicationMailer
+  APPROVED = "people_registration_approved"
+  REJECTED = "people_registration_rejected"
+
+  def approve(person)
+    send_mail(person, APPROVED)
+  end
+
+  def reject(person)
+    send_mail(person, REJECTED)
+  end
+
+  private
+
+  def send_mail(person, content_key)
+    @person = person
+    headers = {bcc: ["mv@sac-cas.ch", person.groups.first.email].compact}
```

Comment:
Die `mv@sac-cas.ch` sollte in einer Konstante (`SacCas::MV_EMAIL`) versorgt werden. Bestehende verwendungen durch konstante ersetzen